### PR TITLE
Update desktop user agent to a modern version

### PR DESCRIPTION
**Pull request by Jacalz**
This pr updates the user agent to a more modern version with latest MacOS Sierra update and the same WebKit and Safari version as in Safari on iOS 🤠
Fixes: https://github.com/brave/browser-ios/issues/876
Fixes: https://github.com/brave/browser-ios/issues/991

### CODE REVIEW

**Comment by Jacalz** [comment]
Would this satisfy you @jhreis or should I revert some things back to how they where before? 🤗

**Comment by Jacalz** [comment]
I have fixed as stated in the review and the pr should be ready for review now 🙂

**Comment by Jacalz** [comment]
Yeah this pr compiles and it also fixes a javascript issue on Github when running with desktop user agent :+1:

**Comment by Jacalz** [comment]
Removed milestone so you can assign this as you would like, this is for me an important fix since it makes JavaScript work in github in desktop tab 🙂

File: brave/src/BraveApp.swift [hunk 1]
```diff
@@ -16,7 +16,7 @@ import Deferred
 private let _singleton = BraveApp()
 
 let kAppBootingIncompleteFlag = "kAppBootingIncompleteFlag"
-let kDesktopUserAgent = "Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Version/5.0 Safari/537.36"
+let kDesktopUserAgent = "Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_12_6) AppleWebKit/603.2.4 (KHTML, like Gecko) Version/10.0 Safari/602.1.31"
```
Comment (jhreis) [inline brave/src/BraveApp.swift]: Just Curious where this user agent came from. I am finding latest safari's to be:

Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/603.3.8 (KHTML, like Gecko) Version/10.1.2 Safari/603.3.8

In every case I have seen Safari's version aligns with the Apple WebKit version.

Comment (Jacalz) [inline brave/src/BraveApp.swift]: This was on iOS 10.3.2, will see if I see the same thing :+1:

Comment (Jacalz) [inline brave/src/BraveApp.swift]: This is what I see on iOS 11 beta 4:
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/604.1.31 (KHTML, like Gecko) Version/10.1.2 Safari/602.1.31

**Pull request merged**